perun-core/src/main/java/cz/metacentrum/perun/core/blImpl/SearcherBlImpl.java

```diff
@@ -23,7 +25,10 @@
 import java.util.Iterator;
 import java.util.List;
 import java.util.Map;
+import java.util.Objects;
```

Comment:
Unused import

Reply:
Removed all unused imports in the class.